types/chart.js/index.d.ts

```diff
@@ -360,6 +361,8 @@ declare namespace Chart {
     }
 
     interface ChartTooltipModel {
+        body: object[];
```

Comment:
This is fine but could you define the interface of the body objects? According to [documentation](https://www.chartjs.org/docs/latest/configuration/tooltip.html#tooltip-model) it should be something like:

```
interface ChartTooltipBodyObject {
  before: string[];
  lines: string[];
  after: string[];
}
```

and then

```
interface ChartTooltipModel {
  bode: ChartTooltipBodyObject[];
  ...
}
```